demoapp/demoapp/UseCases/DateTokenizationViewController.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  DateTokenizationViewController.swift
+//  demoapp
+//
+
+import UIKit
+import VGSCollectSDK
+
+class DateTokenizationViewController: UIViewController {
+    
+    // MARK: - Outlets
+    @IBOutlet weak var containerStackView: UIStackView!
+    @IBOutlet weak var consoleStatusLabel: UILabel!
+    @IBOutlet weak var consoleLabel: UILabel!
+    
+    // MARK: - Properties
+    /// Init VGS Collector
+    var vgsCollect = VGSCollect(
+        id: AppCollectorConfiguration.shared.vaultId,
+        environment: AppCollectorConfiguration.shared.environment
+    )
+    /// VGS UI Elements
+    var dateField = VGSDateTextField()
+    var consoleMessage: String = "" {
+        didSet {
+            consoleLabel.text = consoleMessage
+        }
+    }
+    
+    // MARK: - View life cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        /// Configuration
+        setupUI()
+        setupElementsConfiguration()
+        
+        /// Set custom headers
+        vgsCollect.customHeaders = [
+            "my custome header": "some custom data"
+        ]
+        
+        /// Observe VGSTextFields changes
+        vgsCollect.observeStates = { [weak self] form in
+            
+            self?.consoleMessage = ""
+            self?.consoleStatusLabel.text = "STATE"
+            
+            form.forEach({ textField in
+                self?.consoleMessage.append(textField.state.description)
+                self?.consoleMessage.append("\n")
+            })
+        }
+    }
+}
+
+// MARK: - Private methods
+private extension DateTokenizationViewController {
+    
+    func setupUI() {
+        containerStackView.addArrangedSubview(dateField)
+        let tapGesture = UITapGestureRecognizer(
+            target: self,
+            action: #selector(hideKeyboard)
+        )
+        consoleLabel.addGestureRecognizer(tapGesture)
+        consoleLabel.isUserInteractionEnabled = true
+        view.addGestureRecognizer(tapGesture)
+    }
+    
+    @objc
+    func hideKeyboard() {
+        view.endEditing(true)
+        consoleLabel.endEditing(true)
+    }
+    
+    func setupElementsConfiguration() {
+        
+        /// Start and end dates
+        let startDate = VGSDate(day: 15, month: 1, year: 2010)
+        let endDate = VGSDate(day: 20, month: 12, year: 2025)
+        
+        /// Create configuration
+        let dateFieldConfiguration = VGSDateTokenizationConfiguration(
+            collector: vgsCollect,
+            fieldName: "date_field",
+            datePickerStartDate: startDate,
+            datePickerEndDate: endDate
+        )
+        dateFieldConfiguration.inputDateFormat = .ddmmyyyy
+        dateFieldConfiguration.outputDateFormat = .yyyymmdd
+        dateFieldConfiguration.inputSource = .datePicker
+        dateFieldConfiguration.divider = "-"
+        dateFieldConfiguration.serializers = [
```

Comment:
We can remove serializers from demo app, since outputDateFormat = .yyyymmdd will not work with serializers

Reply:
we can add examples with serializers in docs